### PR TITLE
Adding SSM Example

### PR DESCRIPTION
**Pull request by CarmenAPuccio**
*Issue #, if available:*

*Description of changes:*
Adding example on how to install Fluent Bit and the AWS output plugins on Amazon Linux 2 via AWS Systems Manager

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/fluent-bit/systems-manager-ec2/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* An [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/dev/Welcome.html) bucket to store your Fluent Bit config files along with the AWS output plugins for Fluent Bit.
+
+#### Instructions
+
+An [AWS Sytems Manager Document](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-ssm-docs.html) (SSM Document) defines the actions that Systems Manager performs on your managed instances. Systems Manager includes more than a dozen pre-configured documents that you can use by specifying parameters at runtime. Documents use JavaScript Object Notation (JSON) or YAML, and they include steps and parameters that you specify.
+
+SSM Documents have a concept of plugins that allow you to perform actions to perform on a managed instance. One of those plugins is [aws:DownloadContent](https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-plugins.html) which allows you to download documents and scripts from remote locations including Amazon S3. This is where we will store the Fluent Bit config files along with the AWS output plugins for Fluent Bit. SSM will then be able to download these files onto managed instances when the document is being ran.
```
Comment (PettitWesley) [inline examples/fluent-bit/systems-manager-ec2/README.md]: Grammer nit: "when the document is run."

File: examples/fluent-bit/systems-manager-ec2/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The first step is to get the AWS output plugins for Fluent Bit onto your local machine. You can either run `make` to build each plugin (instructions on are their respective GitHub repositories) or simply just copy them out of the official [AWS for Fluent Bit Docker Image](https://github.com/aws/aws-for-fluent-bit).
+
+```
+docker create -ti --name ssmdemo amazon/aws-for-fluent-bit:latest
+docker cp ssmdemo:/fluent-bit/cloudwatch.so .
+docker cp ssmdemo:/fluent-bit/firehose.so .
```
Comment (PettitWesley) [inline examples/fluent-bit/systems-manager-ec2/README.md]: Add a cp for `/fluent-bit/kinesis.so`, that'll be in the image by the time I merge this.

File: examples/fluent-bit/systems-manager-ec2/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+docker stop ssmdemo
+docker rm ssmdemo
+```
+
+Now you can simply upload the two output plugins along with the [plugins.conf](plugins.conf) file to your S3 bucket. You can reference the aws:downloadContent sections within the SSM Document for [ConfigureFluentBitEC2.yaml](ConfigureFluentBitEC2.yaml) to see how they are referenced. Any other configurations at this point for Fluent Bit would need to be configured and uploaded to the S3 bucket along with a section in the SSM Document to reference it.
```
Comment (PettitWesley) [inline examples/fluent-bit/systems-manager-ec2/README.md]: I think it's better to add an example config and add a section in the SSM document for it. Customers have to customize the config themselves ofc, but including it makes the example more complete.

File: examples/fluent-bit/systems-manager-ec2/README.md (hunk 1 above)
Comment (CarmenAPuccio) [reply examples/fluent-bit/systems-manager-ec2/README.md]: Fixed

**Comment by CarmenAPuccio** [comment]
@PettitWesley - Let me know if this is what you are looking for. The td-agent-bit.conf file sets up CloudWatch output as an example.

File: examples/fluent-bit/systems-manager-ec2/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+docker cp ssmdemo:/fluent-bit/kinesis.so .
+docker stop ssmdemo
+docker rm ssmdemo
+```
+
+Now you can simply upload the AWS output plugins along with the [plugins.conf](plugins.conf) and [td-agent-bit.conf](td-agent-bit.conf) files to your S3 bucket. You can reference the aws:downloadContent sections within the SSM Document for [ConfigureFluentBitEC2.yaml](ConfigureFluentBitEC2.yaml) to see how they are referenced. The example [td-agent-bit.conf](td-agent-bit.conf) for Fluent Bit is configured with the [Foward Input Plugin](https://docs.fluentbit.io/manual/input/forward) with the Output configured to send to Cloudwatch. 
+
+Once that's done, you can then [create the SSM Document](https://docs.aws.amazon.com/systems-manager/latest/userguide/create-ssm-document-cli.html) for [ConfigureFluentBitEC2.yaml](ConfigureFluentBitEC2.yaml) to run against your managed instances.
+
+Any other configurations at this point for Fluent Bit would need to be configured and uploaded to the S3 bucket along with a section in the SSM Document to reference it.
```
Comment (PettitWesley) [inline examples/fluent-bit/systems-manager-ec2/README.md]: nit: "at this point" is unnecessary

File: examples/fluent-bit/systems-manager-ec2/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+```
+docker create -ti --name ssmdemo amazon/aws-for-fluent-bit:latest
+docker cp ssmdemo:/fluent-bit/cloudwatch.so .
+docker cp ssmdemo:/fluent-bit/firehose.so .
+docker cp ssmdemo:/fluent-bit/kinesis.so .
+docker stop ssmdemo
+docker rm ssmdemo
+```
+
+Now you can simply upload the AWS output plugins along with the [plugins.conf](plugins.conf) and [td-agent-bit.conf](td-agent-bit.conf) files to your S3 bucket. You can reference the aws:downloadContent sections within the SSM Document for [ConfigureFluentBitEC2.yaml](ConfigureFluentBitEC2.yaml) to see how they are referenced. The example [td-agent-bit.conf](td-agent-bit.conf) for Fluent Bit is configured with the [Foward Input Plugin](https://docs.fluentbit.io/manual/input/forward) with the Output configured to send to Cloudwatch.
```
Comment (PettitWesley) [inline examples/fluent-bit/systems-manager-ec2/README.md]: "Foward"

**Comment by PettitWesley** [comment]
@CarmenAPuccio Except for two small things in the readme, this looks good to be.

I will merge it after #6

**Comment by CarmenAPuccio** [comment]
@PettitWesley the README should have the two typos fixed. Sorry about that.

**Pull request merged**